jinahub/encoders/image/AudioCLIPImageEncoder/audio_clip/model/clip/clip.py

```diff
@@ -190,8 +190,7 @@ def tokenize(texts: Union[str, List[str]], context_length: int = 77) -> torch.Lo
 
     for i, tokens in enumerate(all_tokens):
         if len(tokens) > context_length:
-            leks = 'albaalbaalbaalbaalbaalbaalbaalbaalbaalbaalbaalbaalbaalbaalbaalba'
-            raise RuntimeError(f"Input {leks} {len(texts[i])} {type(texts[i])} {texts[i]} is too long for context length {context_length}")
+            raise RuntimeError(f"Input {texts[i]} is too long for context length {context_length}")
```

Comment:
do not change this code as it's not ours

Reply:
@tadejsv

Reply:
Answered below